.github/workflows/ci.yaml

```diff
@@ -145,6 +145,15 @@ jobs:
         image: motoserver/moto:4.1.1
         ports:
           - 5053:5000
+      lets_encrypt:
+        image: letsencrypt/pebble:latest
+        container_name: lets_encrypt
```

Comment:
Looks like the container_name is causing the error. I don't think this is a valid field for github actions file